ddtrace/settings/__init__.py

```diff
@@ -1,12 +1,17 @@
-from .._hooks import Hooks
-from .exceptions import ConfigException
-from .http import HttpConfig
-from .integration import IntegrationConfig
-
-
-__all__ = [
+from ddtrace.vendor.debtcollector import deprecate
+_deprecated_names = [
     "ConfigException",
     "HttpConfig",
     "Hooks",
     "IntegrationConfig",
 ]
+
+
+def __getattr__(name):
+    if name in _deprecated_names:
+        deprecate(
+            ("%s.%s is deprecated" % (__name__, name)),
+            removal_version="4.0.0", # TODO: update this to the correct version
+        )
+        return _deprecated_names[name]
+    raise AttributeError("'%s' has no attribute '%s'", __name__, name)
```

Comment:
You may need to run `hatch run lint:fmt` to format this file. You probably need a new line at the end of the file